apps/token-manager/app/src/script.js

```diff
@@ -169,10 +166,7 @@ function updateHolders(holders, changed) {
 
 function loadMaxAccountTokens() {
   return new Promise((resolve, reject) =>
-    app
-      .call('maxAccountTokens')
-      .first()
-      .subscribe(resolve, reject)
+    app.call('maxAccountTokens').subscribe(resolve, reject)
```

Comment:
Maybe we could remove `loadMaxAccountTokens` entirely? Since we can get the value using:

```js
await app.call('maxAccountTokens').toPromise()
```